src/corpus/domain/chapter.ts

```diff
@@ -159,16 +159,33 @@ export class ChapterDisplay {
     return this.textHasDoi ? textIdToDoiString(this.id.textId) : ''
   }
 
+  mapNames(
+    names: readonly Author[] | readonly Translator[]
+  ): { family: string[]; given: string[] }[] {
+    return names.map((_name) => {
+      return {
+        family: _name.name,
+        given: _name.prefix,
+      }
+    })
+  }
+
+  mapAuthorsByRole(role: string): { family: string[]; given: string[] }[] {
```

Comment:
I'd call it `getAuthorsByRole`, I think it would make the intention even clearer! :)